### PR TITLE
Send invites in web user's language

### PR DESCRIPTION
**Pull request by kaapstorm**
[FB 172813](http://manage.dimagi.com/default.asp?172813)

buddy @dannyroberts, ping @snopoke

### CODE REVIEW

File: locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -9250,6 +9250,77 @@ msgstr "Voulez-vous vraiment supprimer cette exportation ?"
...
+"org/."
+msgstr ""
+"Si vous pensez que vous avez reçu cette invitation par erreur, vous pouvez "
+"ignorer cet email. Pour en savoir plus sur CommCareHQ visiter "
+"http://www.commcarehq.org/."
+
+#: corehq/apps/domain/templates/domain/email/domain_invite.html:7
+#: corehq/apps/domain/templates/domain/email/domain_invite.txt:12
+#: venv/lib/python2.7/site-packages/django/contrib/admin/templates/registration/password_reset_email.html:10
+#, fuzzy
```
Comment (emord) [inline locale/fr/LC_MESSAGES/django.po]: I think this fuzzy tag will mean that it will not actually be translated.

**Comment by emord** [comment]
We usually use transifex for translations. I'm not really sure what happens to translations that don't go through transifex (like if they're uploaded correctly, or if they will be overwritten when we pull from there)

**Comment by kaapstorm** [comment]
I don't think it'll be a big deal if we drop the translation; I just thought it would be nice if it got deployed with the French that Pat provided in the FogBugz case. ... In fact, I'll make a new PR, and just push these POs to Transifex instead of to GitHub.

File: locale/fr/LC_MESSAGES/django.po (hunk 1 above)
Comment (kaapstorm) [inline locale/fr/LC_MESSAGES/django.po]: (Apparently, it's when either gettext or a translator thinks a translation should be reviewed.
https://www.gnu.org/software/gettext/manual/html_node/Fuzzy-Entries.html)

**Comment by kaapstorm** [comment]
Grrr. I'll close this PR when I've resolved the Transifex issue in new PR #7256.

**Pull request closed without merging**